spec/services/github_pull_request_review_service_spec.rb

```diff
@@ -1,9 +1,75 @@
 require 'rails_helper'
 
 describe GithubPullRequestReviewService do
+  let(:token) { "blubli" }
+  let(:repository) { create(:repository) }
+  let(:pull_request) { create(:pull_request, repository: repository) }
+
+  let!(:github_reviews_response) do
+    [
+      double(
+        id: 123,
+        user: double(
+          id: 1,
+          login: "milla",
+          name: "Milla Jovovich",
+          email: "milla@jovovich.cl",
+          avatar_url: 'url_to_avatar',
+          html_url: 'url_to_html'
+        )
+      )
+    ]
+  end
+
+  let(:client) { double(:client, pull_requests: true) }
+
   def build(*_args)
     described_class.new(*_args)
   end
 
-  pending "describe what your service does here"
+  before do
+    allow(BuildOctokitClient).to receive(:for).with(token: token).and_return(client)
+
+    allow(client).to receive(:pull_request_reviews)
+      .with(repository.full_name, 1, accept: 'application/vnd.github.thor-preview+json')
+      .and_return(github_reviews_response)
+  end
+
+  describe "#import_all_from_repository" do
+    before do
+      @service = build(token: token)
+    end
+
+    after do
+      @service.import_all_from_repository(repository)
+    end
+
+    it "import from all pr from this repository" do
+      expect(@service).to receive(:import_all_from_pull_request).with(pull_request)
+    end
+  end
+
+  describe "#import_all_from_pull_request" do
+    context "when review doesn`t exist" do
+      it "creates new pull request review" do
+        expect { build(token: token).import_all_from_pull_request(pull_request) }.to(
+          change { PullRequestReview.count }
+            .from(0)
```

Comment:
Se podría cambiar `.from(0).to(1)` por `by(1)` para que el test no depende que la tabla PullRequestReview esté vacía.